po/fr.po

```diff
@@ -13021,208 +13092,297 @@ msgstr[1] ""
 msgid "'|' misplaced, empty tag is not allowed, aborting"
 msgstr "'|' mal placé, un mot-clé vide est interdit, annulation"
 
-#: ../src/libs/tagging.c:1065
+#: ../src/libs/tagging.c:1800
 #, c-format
 msgid "at least one new tagname (%s) already exists, aborting."
 msgstr "au moins un nouveau mot-clé (%s) existe déjà, annulation"
 
-#: ../src/libs/tagging.c:1126
-msgid "copy to entry"
-msgstr "copier l'entrée"
+#: ../src/libs/tagging.c:1900
+msgid "attach tag"
+msgstr "attacher le mot-clé"
 
-#: ../src/libs/tagging.c:1142 ../src/libs/tagging.c:1150
+#: ../src/libs/tagging.c:1908
 msgid "delete tag"
 msgstr "supprimer un mot-clé"
 
-#: ../src/libs/tagging.c:1157
-msgid "rename tag..."
-msgstr "renommer un mot-clé..."
+#: ../src/libs/tagging.c:1918
+msgid "create tag..."
+msgstr "créer un mot-clé..."
 
-#: ../src/libs/tagging.c:1174
+#: ../src/libs/tagging.c:1922
+msgid "edit tag..."
+msgstr "éditer le mot-clé..."
+
+#: ../src/libs/tagging.c:1931
 msgid "rename path..."
 msgstr "renommer un chemin..."
 
-#: ../src/libs/tagging.c:1244
+#: ../src/libs/tagging.c:1938
+msgid "copy to entry"
+msgstr "copier l'entrée"
+
+#: ../src/libs/tagging.c:1962
+msgid "go to tag collection"
+msgstr "aller à la collection liée au mot-clé"
+
+#: ../src/libs/tagging.c:1968
+msgid "go back to work"
+msgstr "retourner au module de mots-clés"
+
+#: ../src/libs/tagging.c:2078
+#, c-format
+msgid "%s"
+msgstr "%s"
+
+#: ../src/libs/tagging.c:2079
+msgid "(private)"
+msgstr "privé"
+
+#: ../src/libs/tagging.c:2105
 msgid "Select a keyword file"
 msgstr "sélectionner un fichier de mot-clé"
 
-#: ../src/libs/tagging.c:1247
+#: ../src/libs/tagging.c:2108
 msgid "_import"
 msgstr "_importer"
 
-#: ../src/libs/tagging.c:1261
+#: ../src/libs/tagging.c:2122
 msgid "error importing tags"
 msgstr "erreur lors de l'import de mot-clé"
 
-#: ../src/libs/tagging.c:1263
+#: ../src/libs/tagging.c:2124
 #, c-format
 msgid "%zd tags imported"
 msgstr "« %zd » mots-clés importés"
 
-#: ../src/libs/tagging.c:1284
+#: ../src/libs/tagging.c:2146
 msgid "Select file to export to"
 msgstr "sélectionne le fichier pour l'export"
 
-#: ../src/libs/tagging.c:1287
+#: ../src/libs/tagging.c:2149
 msgid "_export"
 msgstr "_exporter"
 
-#: ../src/libs/tagging.c:1302
+#: ../src/libs/tagging.c:2164
 msgid "error exporting tags"
 msgstr "erreur lors de l'export de mot-clé"
 
-#: ../src/libs/tagging.c:1304
+#: ../src/libs/tagging.c:2166
 #, c-format
 msgid "%zd tags exported"
 msgstr "« %zd » mots-clés exportés"
 
-#: ../src/libs/tagging.c:1453
+#: ../src/libs/tagging.c:2524
 msgid ""
 "attached tags,\n"
-"double-click to detach"
+"double-click to detach\n"
+"right-click for other actions on attached tag,\n"
+"Ctrl-wheel scroll to resize the window"
 msgstr ""
-"mots-clés attachés,\n"
-"double-clic pour détacher"
+"mots-clés liés,\n"
+"double-clic pour attacher\n"
+"clic-droit pour afficher le menu des actions du mot-clé sélectionné\n"
+"Ctrl+molette souris pour redimensionner la fenêtre"
```

Comment:
No C capital letter here. Same above. And maybe in other places.

Reply:
Normally no other places as I have just respect how the source was written. And as I remember, that was the only ones.

Reply:
Note that you don't have to respect the casing of the standard English strings and when you see such casing do not hesitate to fix the sources directly as I have done just now.